### ISSUE TITLE
CPU example code is broken

### ISSUE THREAD

**Issue by rhyolight** [numenta/nupic#3231]
The example code in `examples/opf/clients/cpu` is not working:

```console
> python cpu.py
Traceback (most recent call last):
  File "cpu.py", line 94, in <module>
    runCPU()
  File "cpu.py", line 50, in runCPU
    model = ModelFactory.create(model_params.MODEL_PARAMS)
  File "/Users/mtaylor/nta/nupic/src/nupic/frameworks/opf/modelfactory.py", line 80, in create
    return modelClass(**modelConfig['modelParams'])
  File "/Users/mtaylor/nta/nupic/src/nupic/frameworks/opf/clamodel.py", line 213, in __init__
    clParams, anomalyParams)
  File "/Users/mtaylor/nta/nupic/src/nupic/frameworks/opf/clamodel.py", line 1184, in __createCLANetwork
    n.initialize()
  File "/Users/mtaylor/nta/nupic/src/nupic/engine/__init__.py", line 717, in initialize
    engine.Network.initialize(self, *args, **kwargs)
  File "/Users/mtaylor/nta/nupic.core/bindings/py/nupic/bindings/engine_internal.py", line 1079, in initialize
    return _engine_internal.Network_initialize(self)
  File "/Users/mtaylor/nta/nupic/src/nupic/regions/SDRClassifierRegion.py", line 259, in initialize
    implementation=self.implementation,
  File "/Users/mtaylor/nta/nupic/src/nupic/algorithms/sdr_classifier_factory.py", line 50, in create
    '"py".' % impl)
ValueError: Invalid classifier implementation ('cpp'). Value must be "py".
```

**Comment by natoromano** [comment]
Fixed by #3223.

**Comment by rhyolight** [comment]
I had to upgrade from capnp 0.5.2 to 0.5.3 to get this example working.
